src/common/Page.ts

```diff
@@ -693,6 +697,17 @@ export class Page extends EventEmitter {
     return this._frameManager.networkManager().setOfflineMode(enabled);
   }
 
+  /**
+   * @param enabled - When `true`, enables offline mode for the page.
```

Comment:
NIT: JSDOC artefact.

Reply:
Actual param is `networkConditions: NetworkConditions`, and not `@param enabled`